src/Sentry/Platforms/Native/buildTransitive/Sentry.Native.targets

```diff
@@ -13,8 +13,8 @@
     <!-- SentryNative.IsEnabled should result in compile-time constant for trimmed applications -->
     <!-- Effectively disabling native library -->
     <RuntimeHostConfigurationOption Include="Sentry.Native.IsEnabled"
-                                    Condition="'$(SentryNative)' == 'false' or '$(SentryNative)' == 'disable'"
-                                    Value="false"
+                                    Condition="'$(FrameworkSupportsNative)' == 'true'"
```

Comment:
suggestion (non-blocking): move this `ItemGroup` just below the modified `PropertyGroup`

Since `ItemGroup`s do get evaluated after `PropertyGroup`s,
I wonder if it makes this project file more clear if we move this `ItemGroup` (that uses `FrameworkSupportsNative`), just below the `PropertyGroup`, that sets the `FrameworkSupportsNative`.

So we also have visually a "set before use" flow.

(non-PR-blocking style suggestion)